src/main/java/io/pravega/connectors/flink/FlinkPravegaWriter.java

```diff
@@ -508,6 +516,7 @@ private void initializeInternalWriter() {
         }
 
         this.clientFactory = createClientFactory(stream.getScope(), clientConfig);
+        LOG.debug("Creating internal writer with write mode: {}", this.writerMode);
```

Comment:
I would suggest this one can be info log

Reply:
fixed